content/home/examples/a-simple-component.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: A Simple Component
+title: ఎ సింపుల్ కంపోనెంట్
 order: 0
 domid: hello-example
 ---
 
-React components implement a `render()` method that takes input data and returns what to display. This example uses an XML-like syntax called JSX. Input data that is passed into the component can be accessed by `render()` via `this.props`.
+రియాక్ట్ కంపోనెంటు `render()` అనే ఒక మెథడ్ని వాడుతుంది, ఇది ఇన్పుట్ డేటా తీసుకుని ఏమి చూపించాలో దాన్ని ఇస్తుంది. ఈ ఉదాహరణ `XML`-లాగ కనిపించే `JSX` అనే సింటాక్స్ వాడుతోంది. ఈ కంపోనెంటుకు పంపించే ఇన్పుట్ డేటాను `this.props` ద్వారా `render()` మెథడ్ చూడగల్గుతుంది.
```

Comment:
`దాన్ని ఇస్తుంది` -> `రిటర్న్ చేస్తుంది`
`చూడగల్గుతుంది` -> I don't have right word in mind but I would suggest to use different word.